### PR TITLE
Fixing bugs and adding more logging

### PR DESCRIPTION
**Pull request by daryadedik**

### CODE REVIEW

File: tests/tests_core/test_experiment.py [hunk 1]
```diff
@@ -63,6 +63,26 @@ def setUp(self):
...
+        self.test_normal_same_zero_std = StatisticalTest(self.data_dummy_zero_std, self.kpi, [], self.variants)
+        self.suite_with_one_test_zero_std = StatisticalTestSuite([self.test_normal_same_zero_std])
+
+        # small dummy data frames with all nan values
+        data_dummy_all_nan = np.array([['index', 'entity', 'variant', 'normal_same', 'normal_shifted'],
+                                       [0, 1, 'A', None, None], [1, 2, 'B', None, None],
+                                       [2, 3, 'A', None, None], [3, 4, 'B', None, None]])
+        self.data_dummy_all_nan = pd.DataFrame(data=data_dummy_all_nan[1:, 1:],
+                                               columns=data_dummy_all_nan[0, 1:]).convert_objects(convert_numeric=True)
+        print(self.data_dummy_all_nan)
```
Comment (shansfolder) [inline tests/tests_core/test_experiment.py]: could you remove the print?

File: expan/core/statistics.py [hunk 2]
```diff
@@ -382,13 +382,15 @@ def compute_statistical_power(mean1, std1, n1, mean2, std2, n2, z_1_minus_alpha)
...
-    :return: statistical power---the probability of a test to detect an effect if the effect actually exists
+    :return: statistical power---the probability of a test to detect an effect if the effect actually exists 
+                                    or -1 if std is less or equal to 0
     :rtype: float
     """
     effect_size = mean1 - mean2
     std = pooled_std(std1, n1, std2, n2)
     if std <= 0.0:
-        raise ValueError('Zero pooled std in compute_statistical_power')
+        logger.warning("Zero pooled std in compute_statistical_power.")
```
Comment (shansfolder) [inline expan/core/statistics.py]: this is an error to me :)

**Review by shansfolder — APPROVED** [review]

File: tests/tests_core/test_experiment.py (hunk 1 above)
Comment (daryadedik) [reply tests/tests_core/test_experiment.py]: @sh yes, forgot to remove!

**Comment by shansfolder** [comment]
Thanks for the coverage increase!

**Pull request merged**